docs/topics/server-overview.md

```diff
@@ -63,7 +63,9 @@ process at JetBrains, is written in 100% Kotlin and has been running in producti
 
 * For a more in-depth introduction to the language, check out the Kotlin documentation on this site and
 [Kotlin Koans](koans.md).
-* Micronaut also has a lot of well-detailed [guides](https://guides.micronaut.io/tags/kotlin.html), showing how you can build microservices in Kotlin.
+* Watch a webinar ["Micronaut for microservices with Kotlin"](https://micronaut.io/2020/12/03/webinar-micronaut-for-microservices-with-kotlin/).
```

Comment:
Maybe it's better to unite in one bullet? 
Because it seems unfair, because other frameworks even don't been mentioned here :)

Reply:
Sure, fixed:) Take a look.